textile/features.textile

```diff
@@ -1207,7 +1207,7 @@ h3(#activation-state-machine). Activation State Machine
 *** @(RSH3g1)@ On event @CalledDeactivate@:
 **** @(RSH3g1a)@ Transitions to @WaitingForDeregistration@.
 *** @(RSH3g2)@ On event @Deregistered@:
-**** @(RSH3g2a)@ Clears all local @DeviceDetails@.
+**** @(RSH3g2a)@ Generates a new pair of device @id@ and @deviceSecret@ as described in "RSH3a2b":#RSH3a2b. Clears all other local @DeviceDetails@.
```

Comment:
This only partially describes the ably-cocoa behaviour, doesn't it? In ably-cocoa, `id` and `deviceSecret` generation are completely decoupled from the `CalledActivate` event, meaning that RSH3a2b and RSH8b would also need to be changed if we wished for the spec to describe the behaviour of ably-cocoa.

Also, same point as https://github.com/ably/specification/pull/182#discussion_r1486624999 re spec point modification.